### ISSUE TITLE
Crashed game on Unity 5.5.2f2 and 5.6.0  when Push?

### ISSUE THREAD

**Issue by buckbim** [Agasper/unity-android-notifications#47]
Hello Everyone.I updated to unity to new version.When I quit game then game notification crashed.
I set Time about 20 secondes push notification.When I click title on bar android device and then game show crashed game.Please suggestion for me to solve this problems.

**Comment by UncleJey** [comment]
having same problem 
```

	readonly static string fullClassName = "net.agasper.unitynotification.UnityNotificationManager";
	readonly static string mainActivityClassName = "com.google.firebase.MessagingUnityPlayerActivity";//"com.unity3d.player.UnityPlayerActivity"; //"com.unity3d.player.UnityPlayerNativeActivity"
	static AndroidJavaClass pluginClass = null;

	static bool CheckPlugin()
	{
#if UNITY_EDITOR
		return false;
#endif
		if (pluginClass == null)
			pluginClass = new AndroidJavaClass(fullClassName);
#if DEBUG_NOTIF
		if (pluginClass == null)
			Debug.LogError("Plugin not found!");
#endif
		return pluginClass != null;
	}

	public static void SendNotification(int id, long delay, string title, string message, Color32 bgColor, bool sound = true, bool vibrate = true, bool lights = true, string bigIcon = null, NotificationExecuteMode executeMode = NotificationExecuteMode.ExactAndAllowWhileIdle)
	{
		if (CheckPlugin())
            pluginClass.CallStatic("SetNotification", id, delay * 1000L, title, message, message, sound ? 1 : 0, vibrate ? 1 : 0, lights ? 1 : 0, bigIcon, "notify_icon_small", bgColor.r * 65536 + bgColor.g * 256 + bgColor.b, (int)executeMode, mainActivityClassName);
	}

```


```
06-07 09:05:37.135: E/Zygote(21559): accessInfo : 0
06-07 09:05:37.492: E/ActivityManager(3649): applyOptionsLocked: Unknown animationType=0
06-07 09:05:37.505: E/AndroidRuntime(7268): FATAL EXCEPTION: main
06-07 09:05:37.505: E/AndroidRuntime(7268): Process: com.foranj.cityship, PID: 7268
06-07 09:05:37.505: E/AndroidRuntime(7268): java.lang.Error: FATAL EXCEPTION [main]
06-07 09:05:37.505: E/AndroidRuntime(7268): Unity version     : 5.5.3f1
06-07 09:05:37.505: E/AndroidRuntime(7268): Device model      : samsung SM-G920F
06-07 09:05:37.505: E/AndroidRuntime(7268): Device fingerprint: samsung/zerofltexx/zeroflte:7.0/NRD90M/G920FXXU5EQCS:user/release-keys
06-07 09:05:37.505: E/AndroidRuntime(7268): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.os.BaseBundle.getString(java.lang.String)' on a null object reference
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at com.google.firebase.MessagingUnityPlayerActivity.onNewIntent(MessagingUnityPlayerActivity.java:46)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1231)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1243)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:3094)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.ActivityThread.performNewIntents(ActivityThread.java:3106)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.ActivityThread.handleNewIntent(ActivityThread.java:3115)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.ActivityThread.-wrap17(ActivityThread.java)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1732)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.os.Handler.dispatchMessage(Handler.java:102)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.os.Looper.loop(Looper.java:154)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at android.app.ActivityThread.main(ActivityThread.java:6724)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at java.lang.reflect.Method.invoke(Native Method)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1520)
06-07 09:05:37.505: E/AndroidRuntime(7268): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1410)
```

**Comment by Agasper** [comment]
Please check the new version

**Comment by UncleJey** [comment]
it was bug in firebase. They are fix it in 4.0.0 version. Thanxx for reply.
